Gemfile.lock

```diff
@@ -9,24 +16,15 @@ GIT
       encryptor
       mongo
 
-GIT
-  remote: git://github.com/Scalarm/scalarm-service_core.git
-  revision: 4448a42f9b17006a1c0defbe1e25d00a34006b3e
+PATH
+  remote: /Users/dkrol/workspace/scalarm/scalarm-service_core
```

Comment:
@dkrol - to chyba należy wycofać (tak jak Gemfile), bo z tego co widzę, to używany jest gem lokalny.